woodwork/serialize.py

```diff
@@ -37,6 +37,16 @@ def typing_info_to_dict(dataframe):
         category_cols = [colname for colname, col in dataframe.ww._schema.columns.items() if col.is_categorical]
         dataframe = dataframe.ww.categorize(columns=category_cols)
     ordered_columns = dataframe.columns
+
+    def get_physical_type_dict(column):
+        type_dict = {}
+        column_dtype = str(column.dtype)
+        type_dict['type'] = column_dtype
```

Comment:
Not very important but this could be:

```python
type_dict = {'type': str(column.dtype)}
```

or
```python
type_dict = {}
type_dict['type'] = str(column.dtype)
```

if we're using the `cat_values` and `cat_dtype` assignments as references for how complex a single can be

Reply:
Cleaned this up a bit.